movies-library/src/components/Counter.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+export class Counter extends React.Component {
```

Comment:
Better go with functional components. Although React team state that class components woun't be removed in any future version of react, but still advice to use functional components in all new projects.

Reply:
In requirements there is info "The component should be written using EcmaScript classes syntax" that's why I used class :)

Reply:
I changed to functional one